server/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+const morgan = require('morgan');
+const parser = require('body-parser');
+// const mongoose = require('mongoose');
```

Comment:
Probably makes sense to take this out until you write something that requires `mongoose` here.

Reply:
Removed.